libparsec/crates/protocol/tests/authenticated_cmds/v4/shamir_recovery_setup.rs

```diff
@@ -0,0 +1,236 @@
+6e74",
+        authenticated_cmds::shamir_recovery_setup::Rep::AuthorIncludedAsRecipient,
+        authenticated_cmds::shamir_recovery_setup::Rep::load
+    );
+}
+pub fn rep_missing_share_for_recipient() {
+    // Generated from Rust implementation (Parsec v3.0.0-b.6+dev 2024-06-06)
+    // Content:
+    //   "status": "missing_share_for_recipient"
+    test_de_serialization!(
+        "81a6737461747573bb6d697373696e675f73686172655f666f725f726563697069656e \
+74",
+        authenticated_cmds::shamir_recovery_setup::Rep::MissingShareForRecipient,
+        authenticated_cmds::shamir_recovery_setup::Rep::load
+    );
+}
+pub fn rep_threshold_greater_than_total_shares() {
+    // Generated from Rust implementation (Parsec v3.0.0-b.6+dev 2024-06-06)
+    // Content:
+    //   "status": "threshold_greater_than_total_shares"
+    test_de_serialization!(
+        "81a6737461747573d9237468726573686f6c645f677265617465725f7468616e5f746f \
+74616c5f736861726573",
+        authenticated_cmds::shamir_recovery_setup::Rep::ThresholdGreaterThanTotalShares,
+        authenticated_cmds::shamir_recovery_setup::Rep::load
+    );
+}
+pub fn rep_timestamp_out_of_ballpark() {
+    // Generated from Rust implementation (Parsec v3.0.0-b.6+dev 2024-06-06)
+    // Content:
+    //   "status": "timestamp_out_of_ballpark"
+    //      "ballpark_client_early_offset": 32.0
+    //      "ballpark_client_late_offset": 32.0
+    //      "client_timestamp": "2009-02-13T23:31:30Z"
+    //      "server_timestamp": "2009-02-13T23:31:30Z"
+
+    let dt = DateTime::from_f64_with_us_precision(1234567890.);
+    let req = authenticated_cmds::shamir_recovery_setup::Rep::TimestampOutOfBallpark {
+        ballpark_client_early_offset: 32.,
+        ballpark_client_late_offset: 32.,
+        client_timestamp: dt,
+        server_timestamp: dt,
+    };
+    test_de_serialization!(
+        "85a6737461747573b974696d657374616d705f6f75745f6f665f62616c6c7061726bbc \
+62616c6c7061726b5f636c69656e745f6561726c795f6f6666736574cb404000000000 \
+0000bb62616c6c7061726b5f636c69656e745f6c6174655f6f6666736574cb40400000 \
+00000000b0636c69656e745f74696d657374616d70d70141d26580b4800000b0736572 \
+7665725f74696d657374616d70d70141d26580b4800000",
+        req,
+        authenticated_cmds::shamir_recovery_setup::Rep::load
+    );
+}
+pub fn rep_require_greater_timestamp() {
+    // Generated from Rust implementation (Parsec v3.0.0-b.6+dev 2024-06-06)
+    // Content:
+    //   "status": "require_greater_timestamp"
+    //   "strictly_greater_than": "2009-02-13T23:31:30Z"
+    //
+    let req = authenticated_cmds::shamir_recovery_setup::Rep::RequireGreaterTimestamp {
+        strictly_greater_than: DateTime::from_f64_with_us_precision(1234567890.),
+    };
+    test_de_serialization!(
+        "82a6737461747573b9726571756972655f677265617465725f74696d657374616d70b5 \
+        7374726963746c795f677265617465725f7468616ed70141d26580b4800000",
```

Comment:
It seems like some payloads are indented and some are not.

Reply:
Thats a known issue that the formatting inside macros is not ideal (see [here](https://github.com/rust-lang/rustfmt/issues/8)). I think I've seen a few fmt unstable options that could help with that.

Anyway, I could have been more careful ^^